compiler/luci/pass/src/CircleQuantizer.cpp

```diff
@@ -313,6 +314,63 @@ bool QuantizeOptionsImpl::query(Algorithm algo)
 
 } // namespace
 
+namespace
+{
+
+bool is_valid_params(loco::Graph *g, LayerParams &lps)
+{
+  // no same name in lps
+  std::unordered_set<std::string> us;
+  for (auto &lp : lps)
+  {
+    if (us.find(lp->name) != us.end())
+      throw std::runtime_error("Duplicate name found in configuration: " + lp->name);
+    us.emplace(lp->name);
+  }
+
+  // all name should be found in graph
+  for (auto &lp : lps)
+  {
+    auto name = lp->name;
+    bool found = false;
+    for (auto node : loco::active_nodes(loco::output_nodes(g)))
+    {
+      auto cnode = loco::must_cast<luci::CircleNode *>(node);
+      if (cnode->opcode() == luci::CircleOpcode::CIRCLEOUTPUT)
+        continue;
+
+      if (cnode->name() == name)
+      {
+        found = true;
+        continue;
+      }
+    }
+    if (not found)
+      return false;
+  }
+  return true;
+}
+
+LayerParams find_valid_params(loco::Graph *g, LayerParamsSet &lpss)
+{
+  // valid condition: there should be only one LayerParams that is OK
+  uint32_t valid_count = 0;
+  LayerParams params;
+  for (auto &lps : lpss)
+  {
+    if (is_valid_params(g, lps))
+    {
+      valid_count++;
+      params = lps;
+    }
+  }
+  if (valid_count == 1)
+    return params;
+  throw std::runtime_error("Invalid item(s) in quantize configuration file.");
```

Comment:
```suggestion
  throw std::runtime_error("Configuration file has layer names (and alternates) that can be mapped in multiple ways. Please update configuration file to have only one valid name mapping.");
```

Reply:
Ah, the above message assumes valid_count > 0.

The mssage has to be changed for the `valid_count ==0` case.